libs/atqr/domain/src/lib/deadline-interface.ts

```diff
@@ -1,6 +1,7 @@
 import { Guid } from "@tokilabs/lang";
+import { Challenge } from "..";
 
 export interface DeadlineInterface {
-  challengeId: Guid;
+  id: Guid;
```

Comment:
This branch and PR are about the challenge entity test, deadline interface changes should not have been made here. Please stash and then remove these changes.